src/kicker/main.c

```diff
@@ -317,21 +332,22 @@ ISR(PCINT0_vect) {
  * (When CS00 is one, and CS01 is 0, no prescale. We don't use this)
  */
 ISR(TIMER0_COMPA_vect) {
```

Comment:
This seems like alot of code to call every 80 processor cycles. Do we need that granular timing.

Reply:
Probably not, but all this chip has to do during the kick cycle is time the kicking. You don't need ballsense or voltage monitoring during that time.